joining.html

```diff
@@ -0,0 +1,217 @@
+    <h2>
+      <a name="joining" href="#joining">#</a>
+      Joining In Game
+    </h2>
+    <p>
+      If you're invited to join, you can now apply to join <b>Nobody in
+      Local</b> in game. A director will give your application a final check,
+      then accept your in-game application. Feel free to wait to confirm joining
+      in game if you need time to move assets or similar.
+    </p>
+
+    <h2>
+      <a name="onboarding" href="#onboarding">#</a>
+      Onboarding
+    </h2>
+    <p>
+      Once you've joined in game, head on over to <a href="onboarding.html">
+      onboarding</a> to get set up!
+    </p>
+
+    <h2>
+      <a name="questions" href="#questions">#</a>
+      Questions You Might Have
+    </h2>
+    <h3>
+      <a name="why-sp-max" href="#why-sp-max">#</a>
+      Why is there an SP maximum?
+    </h3>
+    <p>
+      Short answer: because we want SOUND to be a group focused on teaching
+      and learning.
+    </p>
+    <p>
+      Long answer: while we know that players of every SP level have things to
+      learn in Eve, our past experience has been that taking a lot of high-SP
+      players makes it more difficult for low-SP players to ask questions, and
+      can lead to a "bittervet" atmosphere of negativity about the game. That
+      isn't fun for anyone.
+    </p>
+    <p>
+      If you're over the 15M SP maximum, don't worry - you can still join, but
+      there's an extra step. Hop into our public channel or join a couple of
+      our regular public fleets, and try to get to know a couple of existing
+      members of the corp. We're looking for culture fit here a lot more than
+      pilot skill - it's fine if you're bad at the game, as long as you have a
+      good attitude about it.
+    </p>
+    <h3>
+      <a name="why-sp-min" href="#why-sp-min">#</a>
+      Why do I need to run PvE missions to join a PvP corp?
+    </h3>
+    <p>
+      Short answer: because it teaches you a lot of basic game mechanics.
+    </p>
+    <p>
+      Long answer: while we definitely welcome players who are brand new to
+      Eve, doing a small part of the game's learning curve on your own before
+      you join will help make sure that you have a solid foundation and enough
+      interest in Eve to stick with it for the long haul. If you need help with
+      any of the missions, feel free to ask for help in our in-game public
+      channel; there will usually be people there who can offer advice and
+      perhaps even assistance with the missions.
+    </p>
+    <p>
+      The career missions also do a good job of introducing you to some of the
+      many professions in Eve, which will help guide you towards what you
+      might want to learn more about.
+    </p>
+
+    <h3>
+      <a name="why-trial-fleet" href="#why-trial-fleet">#</a>
+      Why do I need to go on a fleet before I even join?
+    </h3>
+    <p>
+      Short answer: because we want to make sure you aren't annoying on
+      voice comms.
```

Comment:
Would suggest changing to "we want to see what you're like on voice comms."

Reply:
Done!